File: Modul 4. Loop/PRAK402/PRAK402-23110817120003-NatalieGraceKatiandagho.c
#include <stdio.h>

void main()
{
    int batas;
    int x;
    scanf("%d", &batas);

    for (x = 1; x <= batas; x++)
    {
        if (x % 2 != 0)
        {
            printf("%d ", x);
        }
    }

    printf("\n");

    for (x = batas; x > 0; x--)
    {
        if (x % 2 == 0)
        {
            printf("%d ", x);
        }
    }
}

File: Modul 4. Loop/PRAK402/PRAK402-23110817120003-NatalieGraceKatiandagho.py
batas = int(input())

for x in range(1, batas + 1):
    if x % 2 != 0:
        print(f"{x} ", end="")

print()

for x in range(batas, 0, -1):
    if x % 2 == 0:
        print(f"{x} ", end="")